src/style/colors.ts

```diff
@@ -131,6 +131,15 @@ export class Colors {
     return ColorName.name(color)[1];
   }
 
+  getSystemColorByHex(colorsOptions: string[], colorValue: string) {
+    const color = colorStringValue(colorValue);
+    const isColorFound = (c: string) => c.toString() === color;
+    const system_color = _.findKey(this, isColorFound);
```

Comment:
Now when you moved all the logic to the `return`, you can add the callback inside the `findKey` itself: 
```
const system_color = _.findKey(this, c => c.toString() === color);
```